app/src/main/res/layout/activity_editcustomerinfo.xml

```diff
@@ -0,0 +1,324 @@
+                android:padding="@dimen/dp_10"
+                app:srcCompat="@drawable/ic_left_arrow"
+                android:onClick="@{() -> viewModel.onCancelClick()}"
+                tools:ignore="@null"/>
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="@dimen/dp_10"
+                android:layout_marginLeft="@dimen/dp_40"
+                android:layout_marginRight="@dimen/dp_10"
+                android:layout_marginStart="@dimen/dp_40"
+                android:gravity="end"
+                android:text="@string/update"
+                android:onClick="@{() -> viewModel.onUpdateClick()}"
+                android:textColor="@color/color_white"
+                android:textSize="@dimen/sp_15"/>
+        </LinearLayout>
+
+        <ProgressBar
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:visibility="@{viewModel.progressBarVisibility == false ? View.GONE : View.VISIBLE}"
+            />
+
+        <android.support.v4.widget.NestedScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical">
+
+                    <RelativeLayout
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center_horizontal|center_vertical"
+                        >
+
+                        <android.support.design.widget.FloatingActionButton
+                            android:layout_width="@dimen/dp_40"
+                            android:layout_height="@dimen/dp_40"
+                            android:layout_centerHorizontal="true"
+                            android:layout_centerVertical="true"
+                            android:layout_margin="@dimen/dp_16"
+                            android:onClick="@{() -> viewModel.onUpdateAvatarButtonClick()}"
+                            app:fabSize="normal"
+                            app:srcCompat="@drawable/ic_edit"
+                            />
+
+                        <ImageView
+                            android:layout_width="@dimen/dp_100"
+                            android:layout_height="@dimen/dp_100"
+                            android:contentDescription="@null"
+                            android:padding="@dimen/dp_10"
+                            android:src="@drawable/ic_customer"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.